### ISSUE TITLE
Overwatch Report Abuse

### ISSUE THREAD

**Issue by RainDota** [ValveSoftware/Dota2-Gameplay#22304]
I'm a several year player that was sunk into the 6k behavior score realm by two separate party queue stacks. I've been here now for about a year, grinding out turbo trying to get up into a good score. All it takes is one series of reports by 3/4/5 players stacked with or against you, that want to troll and flame, and you're back to lower than where you started. I'm sick of it. Fix, or get rid of the system entirely. It's being abused daily

**Comment by vanndano** [comment]
Yes the system sucks and devs seem to be happy making people play yo just grind Behavior score instead of MMR and not address report abuse. My recommendation is honestly dont communicate if you are gonna keep playing, many players simply report anyone that engages. 

Also dont grind in turbo, why? Because devs significantly reduced the Behavior score uou could win if you had 0 report and 30+ commends. Reports count just as much as other game modes, so the more you play the higher the chances even noise reports will drag you back.  much better to do it in unranked. 

Say you want to go back to 10k bs. Its about 180 BS gained in turbo grimding and a bit kore than double that in regular matchmaking. The way the system stands rn youd have to play 300 turbo matches or 150 regular ones without a single report. If you arent having fun and you get repeatedly reported without griefing ask yourself if its worth your time. My suggestion is: quit until this get fixed or addressed. 

Devs have been making dumb decisions that weaponize reports and actual griefers get away with a slap in the wrist, none of it conveys people to behave better and sours the game, even at 12k behavior score.

**Comment by thedarknomad** [comment]
>  All it takes is one series of reports by 3/4/5 players stacked with or against you, that want to troll and flame, and you're back to lower than where you started.

Reports from parties count as 1 ticket. It's been like that for years. So if they are 4 and all report you, you only have 1 ticket against you. Turbo is not the place to get better scores, it's way more toxic.
